lib/commands/travis-visual-acceptance.js

```diff
@@ -91,11 +100,14 @@ module.exports = {
     process.env.NO_ASSERT = true
 
     var prNumber = process.env.TRAVIS_PULL_REQUEST
+
     if (prNumber !== false && prNumber !== 'false' && process.env.VISUAL_ACCEPTANCE_TOKEN) {
       return runCommand(this, 'br').then(buildReport, function (params) {
```

Comment:
Need to add the if statement to the first param as well.
`.then(buildReport, ....)` ->
```
.then( function (params){
        if (options.msgOn === options.msgVar) {
           return buildReport(params)
        }
}, ....
```